DecipherNow/gm-fabric-dashboard
# 963 chart xaxis gets cut off

Resolves #963 

## Proposed Changes

  - added flex-basis auto since IE has issues with interpreting flex box basis and defaults to 1 in all instance which can cause issues. Specifying auto resolves this. 
  
## How to Test

  - Navigate to instance summary page for ether JVM or GO and make sure chart at the bottom of summary page isn't cut off. 
 
This fix has been verified in:
Safari 11
IE 11
Edge 16, 15, 14
Chrome 62
FF 52
